intelmq/bots/experts/jinja/expert.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-FileCopyrightText: 2021 Birger Schacht
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from intelmq.lib.bot import Bot
+
+import pathlib
+import os
+
+try:
+    from jinja2 import Template, TemplateError
+except ImportError:
+    Template = None
+
+
+class JinjaExpertBot(Bot):
+    """
+    Modify the message using the Jinja templating engine
+    Example:
+    fields:
+      output: The provider is {{ msg['feed.provider'] }}!
+      feed.url: "{{ msg['feed.url'] | upper }}"
+      extra.somejinjaoutput: file:///etc/intelmq/somejinjatemplate.j2
+    """
+
+    fields: dict = {}
+
+    def init(self):
+        if not Template:
+            raise ValueError("Library 'jinja2' is required, please install it.")
+
+        for field, template in self.fields.items():
+            if template.startswith("file:///"):
+                templatefile = pathlib.Path(template[7:])
+                if templatefile.exists() and os.access(templatefile, os.R_OK):
+                    self.fields[field] = templatefile.read_text()
+                else:
+                    raise ValueError(f"Jinja Template {templatefile} does not exist or is not readable.")
+
+        for field, template in self.fields.items():
+            try:
+                self.fields[field] = Template(template)
+            except TemplateError as msg:
+                raise ValueError(f"Error parsing Jinaj Template for '{field}': {msg}")
+
+    def process(self):
+        msg = self.receive_message()
+
+        for field, template in self.fields.items():
+            msg.add(field, template.render(msg=msg), overwrite=True)
```

Comment:
```suggestion
            msg.add(field, template.render(msg=msg), overwrite=self.overwrite)
```
can we haz parameter for this? (the overwrite parameter is already used on various similar bots)

Reply:
Sure- I wasn´t sure about that because the parameter then would apply to multiple fields at once